test/immutable-items-strict-spec.js

```diff
@@ -0,0 +1,862 @@
+var testHelper = require('../etc/test-helper.js');
```

Comment:
As noted in the pull request description, this file borrows from `immutable-items-spec.js`.